src/events/order-confirmed-consumer.ts

```
@@ -0,0 +1,36 @@
+  datacontenttype: string;
+  data: {
+    id: string;
+    status: string;
+  };
+}
+
+export class OrderConfirmedConsumer extends BaseConsumer<OrderConfirmedEvent> {
+  public topic = "order.created";
+  constructor(consumer: any) {
+    super(consumer);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)